gpt_index/readers/file/markdown_parser.py

```diff
@@ -0,0 +1,90 @@
+
```

Comment:
nit: add docstring to top of file

Also bigger note, now that you've defined this parser, you can use this parser as part of SimpleDirectoryReader. Take a look at DEFAULT_FILE_EXTRACTOR in gpt_index.readers.file.base, you can add this as a parser to parse '.md' files.